### ISSUE TITLE
[TNT/Scorebox/10v10] Retro Encabulator

### ISSUE THREAD

**Issue by OhPointFive** [OvercastCommunity/CommunityMaps#773]
### Checklist

- [x] I have [pruned](https://pgm.dev/docs/guides/packaging/pruning-chunks) the map.
- [x] I have agreed with assigning the CC BY-SA license to this map, as mentioned in the [README](https://github.com/OvercastCommunity/CommunityMaps#readme).
- [x] I have read and understood the  [map submission guidelines](https://oc.tc/submissions).
- [x] I have created an XML file.
- [x] I have created a map image.
- [x] I have uploaded the map zip file to a file sharing service.
- [x] The map has been tested locally to make sure it works.

### Map Name

Retro Encabulator

### Gamemode & Map Description

Teams must battle, build, and blast past their opponents to get into the scorebox, all while keeping the opposing team out of theirs. Most goals after 12 minutes wins.

### Team Sizes

2 teams of 10

### Screenshots
[Screenshots](https://imgur.com/a/KQ9JgWU)

### XML

XML is included in the world download

### Map Image

![map](https://github.com/OvercastCommunity/CommunityMaps/assets/41559602/f792b77b-c18a-4f31-8fd5-3dc68719c2c0)

### Map Download

[Retro Encabulator.zip](https://github.com/OvercastCommunity/CommunityMaps/files/11936026/Retro.Encabulator.zip)

The map is not pruned since I couldn't get MCASelector to work

**Comment by arcadeboss** [comment]
People seemed to enjoy this during play testing today.  GJ.

**Comment by OhPointFive** [comment]
[Retro Encabulator.zip](https://github.com/OvercastCommunity/CommunityMaps/files/11972807/Retro.Encabulator.zip)

Based on feedback, I gave slightly more protection to the spawn armour, slightly spread out middle islands, and fixed spawn breaking & block glitching exploits. Haven't made any big changes since I haven't seen a test myself yet.

**Comment by arcadeboss** [comment]
I played at test yesterday, was fun.

Take with a grain of salt, but my suggestions:
- Make the 1 block wide gap to get from spawn area to main map wider.   I couldn't find it the first time I played.  Now, I know but still feels too small
- Add another stack of TNT.   Was defending with TNT, but realized I would soon run out.  If there are TNT supply chests somewhere, then it's just a knowledge issue on my part.
